src/components/message/Message.tsx

```diff
@@ -19,35 +19,37 @@ interface MessageProps {
 const Message: FC<MessageProps> = ({ message }) => {
   const { userId: myId } = useAppSelector((state) => state.appMain)
 
-  const { author, messageContent, authorRole } = message
-  const { _id, firstName, lastName, photo, createdAt } = author
-
+  const { author, text, authorRole, createdAt } = message
+  const { _id, photo } = author
   const isMyMessage = myId === _id
+  const userURL = createUrlPath(authRoutes.userProfile.path, _id, {
+    authorRole
+  })
+
+  const handleLinkClick = (e: MouseEvent<HTMLAnchorElement>) => {
+    e.stopPropagation()
+  }
+
+  const date = getFormattedDate({
+    date: createdAt,
+    options: { hour: '2-digit', minute: '2-digit' }
+  })
+
+  const avatar = !isMyMessage && (
+    <Link onClick={handleLinkClick} to={userURL}>
+      <Avatar
+        src={photo && `${import.meta.env.VITE_APP_IMG_USER_URL}${photo}`}
```

Comment:
i think it would be better to move this import to separate variable and then put here